docs/src/get-started/monitor-and-troubleshoot/log.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Git log
```

Comment:
as it seems that rest of the page is not related to git, title should be `Platform log`, isn't it?

Reply:
The idea is to keep the sections linked to Git commands to fit with the idea of GitOps. Most of the commands aren't related to `platform log` either. :shrug: